src/test/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSourceTraitsTest.java

```diff
@@ -93,7 +93,7 @@ public void repositoryUrl() throws Exception {
         GitHubSCMSource instance = load();
 
         assertThat(instance.getId(), is("e4d8c11a-0d24-472f-b86b-4b017c160e9a"));
-        assertThat(instance.getApiUri(), is(GitHubSCMSource.GITHUB_URL));
+        assertThat(instance.getApiUri(), is("https://api.github.com"));
```

Comment:
nit: I would change this back to `GitHubSCMSource.GITHUB_URL` like it was before.

Reply:
Good catch. Done. :)